<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GestiÃ³n de Clientes</string>
  </property>
  <widget class="QPushButton" name="botonCliente">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonRegresar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewClientes">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>461</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
